Cargo.toml

```
@@ -32,7 +32,7 @@ rust-version = "1.73"
 all-features = true
 
 [dependencies]
-arrow = { version = "53.1.0", features = ["prettyprint", "chrono-tz"] }
+arrow = { version = "54.2.1", features = ["prettyprint", "chrono-tz"] }
```

```suggestion
arrow = { version = "54.0.0", features = ["prettyprint", "chrono-tz"] }
```

no need to be strict as far as I can tell